cabal.project

```diff
@@ -38,6 +38,7 @@ packages:
   eras/babbage/test-suite
   eras/conway/impl
   eras/conway/test-suite
+  eras/dijkstra
```

Comment:
I think it's fine to put the project directly into `eras/dijkstra`, since we don't want to have the `cardano-ledger-<era>-test` packages anymore.

Reply:
Yes, that is fine, especially since that is the ultimate goal for all of the previous eras. :+1: